File: jewellery_erpnext/jewellery_erpnext/web_form/exit_employee/exit_employee.js
frappe.ready(function() {
	// bind events here
	frappe.web_form.on('employee', (field, value) => {
		frappe.call({
			method: 'jewellery_erpnext.jewellery_erpnext.web_form.exit_employee.exit_employee.get_context',
			args: {
				'employee' :value
			},
			callback: (data) => {
				if (data.message) {
					console.log(data.message);
					frappe.web_form.set_value(['employee_name'], [data.message[0]['employee_name']])
					frappe.web_form.set_value(['department'], [data.message[0]['department']])
					frappe.web_form.set_value(['designation'], [data.message[0]['designation']])
				}
			}
		})
	})
})

File: jewellery_erpnext/jewellery_erpnext/web_form/exit_employee/exit_employee.py
import frappe

@frappe.whitelist()
def get_context(employee):
	employee_data = []
	for i in frappe.get_list('Employee', ['name','employee_name','department','designation']):
		if i['name'] == employee:
			employee_data.append(i)
			break
	return employee_data
	
